code/modules/antagonists/superheroes/spawners.dm

```diff
@@ -0,0 +1,286 @@
+	roundstart = FALSE
+	death = FALSE
+	show_flavour = FALSE
+	anchored = TRUE
+	density = FALSE
+	mob_species = /datum/species/human
+	short_desc = "You are a superhero."
+	flavour_text = "You are a superhero aboard the OwlSkip. Help your fellow superheroes and catch those peskys villains!"
+	assignedrole = "Superhero"
+	outfit = /datum/outfit/superhero
+	var/hero_role = "Coder's Fuckup"
+
+/obj/effect/mob_spawn/human/superhero/Initialize()
+	. = ..()
+	notify_ghosts("[assignedrole]s are here! Click this button to become one.", source = src, action=NOTIFY_ATTACK, flashwindow = FALSE)
+
+/obj/effect/mob_spawn/human/superhero/Destroy()
+	new/obj/structure/fluff/empty_cryostasis_sleeper(get_turf(src))
+	return ..()
+
+/obj/effect/mob_spawn/human/superhero/special(mob/living/new_spawn)
+	var/datum/antagonist/superhero/hero = new()
+	hero.hero_role = hero_role
+	new_spawn.mind.add_antag_datum(hero)
+	hero.greet()
+	message_admins("[ADMIN_LOOKUPFLW(new_spawn)] has been made into a [assignedrole].")
+	log_game("[key_name(new_spawn)] was spawned as a [assignedrole]л.")
+
+/obj/effect/mob_spawn/human/superhero/villain
+	short_desc = "You are a supervillain."
+	flavour_text = "You are a supervillain aboard the Dark Mothership. Help your fellow villins and catch those heroes!"
+	assignedrole = "Supervillain"
+	outfit = /datum/outfit/superhero/villain
+
+/obj/effect/mob_spawn/human/superhero/villain/special(mob/living/new_spawn)
+	var/datum/antagonist/supervillain/villain = new()
+	villain.hero_role = hero_role
+	new_spawn.mind.add_antag_datum(villain)
+
+//Heroes
+
+/obj/effect/mob_spawn/human/superhero/buzzon
+	name = "BuzzOn's cryostasis sleeper"
+	desc = "A cryostasis sleeper containing BuzzOn, the creator of the robotic bee suit. It smells of honey and flowers."
+
+	outfit = /datum/outfit/superhero/buzzon_nude/spawner
+	hero_role = "BuzzOn"
+
+/obj/effect/mob_spawn/human/superhero/ianiser
+	name = "Ianiser's cryostasis sleeper"
+	desc = "A cryostasis sleeper containing Ianiser, the great electrofurry. It has a lot of dirt on it and is sparking a little."
+
+	outfit = /datum/outfit/superhero/ianiser_spawner
+	hero_role = "Ianiser"
+
+/obj/effect/mob_spawn/human/superhero/owlman
+	name = "Owlman's cryostasis sleeper"
+	desc = "A cryostasis sleeper containing Owlman, the leader of the superhero team. It has a lot of scratches on it."
+
+	outfit = /datum/outfit/superhero/owlman_nude
+	hero_role = "Owlman"
+
+//Villains
+
+/obj/effect/mob_spawn/human/superhero/villain/skeledoom
+	name = "SkeleDoom's cryostasis sleeper"
+	desc = "A cryostasis sleeper containing an edgy teen looking like a skeleton. It's probably SkeleDoom."
+
+	outfit = /datum/outfit/superhero/villain/skeledoom_nude/spawner
+	hero_role = "SkeleDoom"
+
+/obj/effect/mob_spawn/human/superhero/villain/nekometic
+	name = "Nekometic's cryostasis sleeper"
+	desc = "A cryostasis sleeper containing Nekometic. Is he a real catboy or just a pervert wearing an anime skirt?"
```

Comment:
this is explicitly fetish content, not allowed under repository rules, make a new superhero design